e2e/impl/wallet_backend.go

```diff
@@ -51,7 +51,34 @@ func (ctx *WalletBackendContext) TheWalletBackendNodeIsRunning() error {
 	}
 	redisAddr := fmt.Sprintf("%v:6379", redisIP)
 
-	transactionsPersistanceSpec, err := cluster.TransactionsPersistanceSpec(ctx.globalCtx.nodeSuffix, rpcAddr, redisAddr)
+	nsqlookupdSpec, err := cluster.NsqlookupdSpec(ctx.globalCtx.nodeSuffix)
```

Comment:
I think this `TheWalletBackendNodeIsRunning` function is getting way to long. Would be nice to refactor somehow.